site/public/js/solution-ta-notes.js

```diff
@@ -0,0 +1,61 @@
+function updateSolutionTaNotes(gradeable_id, component_id) {
+  let data = {
+    solution_text: $(`#textbox-solution-${component_id}`).val().trim(),
+    component_id,
```

Comment:
what is this `component_id` used for, should it have a value associated with it ?

Reply:
Its a parameter needed for updating the solution notes...It is associated with the value of `component_id`. Basically in JS objects if you have both key and value names exactly the same you can use this [shorthand syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#Property_definitions). I believe we are using this syntax in some JS files already.